db/migrate/20220428135032_rename_id_to_show_id.rb

```diff
@@ -0,0 +1,5 @@
+class RenameIdToShowId < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :shows, :id, :show_id
```

Comment:
Ditto the above comment, it's not a big deal but I just thought that since these migrations haven't been run yet then it's a good opportunity to avoid creating unnecessary migrations